external_equipment/sam_joy_controllers/sam_joy_controllers/sam_joy_xbox/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>sam_joy_xbox</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="axbr@kth.se">axel</maintainer>
+  <license>MIT</license>
+
+  <test_depend>ament_copyright</test_depend>
+  <test_depend>ament_flake8</test_depend>
+  <test_depend>ament_pep257</test_depend>
+  <test_depend>python3-pytest</test_depend>
+  <depend>rclpy</depend>
+  <depend>sensor_msgs</depend>
+  <depend>std_msgs</depend>
+  <depend>sam_msgs</depend>
+  <depend>smarc_control_msgs</depend>
```

Comment:
Remove. No smarc package is installable through rosinstall, they are _all_ in the smarc2 repo.